bindings/pyroot/src/RootWrapper.cxx

```diff
@@ -589,15 +544,6 @@ PyObject* PyROOT::CreateScopeProxy( const std::string& scope_name, PyObject* par
    const std::string& lookup = parent ? (scName+"::"+name) : name;
    Cppyy::TCppScope_t klass = Cppyy::GetScope( lookup );
 
-   if ( ! (Bool_t)klass || Cppyy::GetNumMethods( klass ) == 0 ) {
```

Comment:
Hi @yamaguchi1024 , I understand this supersedes [one](https://github.com/root-project/root/pull/1547/commits/499c2a600eadf04bee67928e706a8c98f6929b03) of the commits in #1547?

Reply:
Yes, that's both memory and performance efficient. We also got green light from Wim. I suspect that's the right thing to do.